scripts/rebase.sh

```diff
@@ -240,21 +241,25 @@ get_release_images() {
 
 # Updates the image digests in pkg/release/release*.go
 update_images() {
-    if [ ! -f "${STAGING_DIR}/release.txt" ]; then
+    if [ ! -f "${STAGING_DIR}/release_amd64.json" ]; then
```

Comment:
Don't you have to check also if release_arm64.json exists?

Reply:
They should always come together, but better to be safe. Fixed this.